setup.py

```diff
@@ -86,6 +87,11 @@ def make_ezipfile(base_name, base_dir, verbose=0, dry_run=0, **kwargs):
         "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.3",
```

Comment:
Many popular packages don't support Python 3.3 anymore (see `requests`, `cryptography`, etc).  [It also recently went end of life](https://www.python.org/dev/peps/pep-0398/#x-end-of-life).

For scapy, I would think 2.7,  3.4+ would probably be appropriate.

Reply:
Is there any good reason why we'd drop Python 3.3 support?

The only reason I do not want Scapy to support Python 3.2 is that the explicit unicode syntax (`u"test"`) is invalid, which makes it harder to support both Python 2 and 3.

Reply:
It would be fine to keep supporting 3.3. I just brought it up based on what I've seen. It saves CI builds if each python version is explicitly tested (which I'd recommend). If a bug in the interpreter is fixed upstream that affects scapy, it wouldn't make it back to 3.3 any longer.

I think the user share of 3.3 is just about 0% last I read about it. Just thought it would be helpful to suggest this.

Reply:
Python 3.3 is EOL so unless there's a specific need I wouldn't bother. And you're right, the share is indeed just about 0%, or in fact 0.1% of pip installs from PyPI for the last month:

```
$ pypinfo --percent --pip "" pyversion
python_version percent download_count
-------------- ------- --------------
2.7              77.6%    492,906,756
3.6               9.0%     57,412,072
3.5               7.7%     48,837,128
3.4               4.3%     27,105,552
2.6               1.3%      8,116,050
3.3               0.1%        754,380
3.7               0.1%        378,511
3.2               0.0%         50,201
None              0.0%         21,257
2.8               0.0%            188
3.1               0.0%            134
```

Reply:
I am fine with keeping 3.3 as long as there is not weird trick on our side, and that this doe not prevent us to use syntax from older versions as 2.4 did.

This PR shows that supporting 3.3 does not cause huge problems.